src/main/webapp/script.js

```diff
@@ -104,20 +104,36 @@ function createParagraphElement(text) {
 // This function returns a message based on sentiment scores
 function getScoreResponse(score) {
   // Messages based on score ranges
-  //TODO: update messages to be something more readable
-  const messages = ["Negative", "Neutral", "Positive"];
+  //TODO: updated messages to a series of good/bad/netural messages. Make it a random selection.
+  const positiveMessages = [
+    "Liking the positive energy!!",
+    "That should make someone smile.",
+    "Killing them with kindness.",
+  ];
+  const neutralMessages = [
+    "Congrats, You achieved a neutral response",
+    "This text shouldn't ruffle any feathers.",
+    "You're like the swiss, Neutral.",
+  ];
+  const negativeMessages = [
+    "Congrats, this text is sort-of negative. If you meant it.",
+    "Sometimes you need to get a somber tone across.",
+    "Let the frustration out. it's ok.",
```

Comment:
Maybe not too appropriate?

Reply:
Are you referring to the last message or all three?

Reply:
Sounds good. I'll update that one.

Reply:
Just updated it.